src/jit/function-builder.cc

```diff
@@ -25,6 +25,7 @@
 #include <cmath>
 #include <limits>
 #include <type_traits>
+#include <iostream>
```

Comment:
Why do we need this? I assume this is a remnant of some `printf`-based debugging that you forgot to remove?